packages/zod-firebase/src/query/query-specification.ts

```
@@ -1,33 +1,76 @@
   limit as firestoreLimit,
+  limitToLast as firestoreLimitToLast,
   orderBy as firestoreOrderBy,
   type OrderByDirection,
   type Query,
   query as firestoreQuery,
   type QueryConstraint,
+  startAfter as firestoreStartAfter,
+  startAt as firestoreStartAt,
   where as firestoreWhere,
   type WhereFilterOp,
 } from '@firebase/firestore'
```

ESLint

[error] Unable to resolve path to module '@firebase/firestore'.

(import-x/no-unresolved)